tests/async/tasync_misc.nim

```diff
@@ -74,3 +74,10 @@ block: # issue #13899
       result = true
   doAssert waitFor testFooSucceed()
   doAssert waitFor testFooFails()
+
+block: # 9313
```

Comment:
`s/9313/issue #9313` so that it's easier to search than a raw number, which would give tons of false positives.

ditto elsewhere.

(ideally we'd have that convention spelt out in https://nim-lang.github.io/Nim/nep1.html to avoid those discussions)

Reply:
@timotheecour I tried to follow whatever namings are already there in files but okay

Reply:
thanks for the change; as noted in  #14796 following surrounding code leads to dialects that survive ;-)